src/utils/getTimeDifference.js

```diff
@@ -0,0 +1,24 @@
+export const getTimeDifference = previousTimestamp => {
+  const msPerMinute = 60 * 1000;
+  const msPerHour = msPerMinute * 60;
+  const msPerDay = msPerHour * 24;
+  const msPerMonth = msPerDay * 30;
+  const msPerYear = msPerDay * 365;
+
+  const currentTimestamp = Date.now();
+  const elapsed = currentTimestamp - previousTimestamp * 1000;
+
+  if (elapsed < msPerMinute) {
```

Comment:
`switch` may be better here

Reply:
swtch tyo comparision ma mildian rahecha